ui/scss/component/_claim-list.scss

```diff
@@ -1204,6 +1229,51 @@
   }
 }
 
+.protected-content-overlay {
+  @include font-sans;
+
+  background-color: rgba(0, 0, 0, 0.5);
+
+  position: absolute;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  z-index: 5;
+  width: 100%;
+  height: 100%;
+  -webkit-backdrop-filter: blur(2px);
+  backdrop-filter: blur(4px);
+  color: white;
+
+  .protected-content-overlay--subheader{
+    font-size: 21px;
+  }
+
+  .icon.icon--Lock {
+    overflow: visible;
+    stroke-width: 3;
+    width: 100px;
+    height: 100px;
+    margin-bottom: var(--spacing-xxxs);
+    padding: var(--spacing-s);
+    margin-top: -54px;
```

Comment:
Hardcoded values without justification/explanation again.

We've been through this ... it almost always will not work for all screens and zoom levels.

<img width="282" alt="image" src="https://user-images.githubusercontent.com/64950861/193552433-e9bd7b00-5f44-4783-9972-f50f649ea1df.png">

Reply:
Why did you even change it Anthony? You're breaking my stuff instead of fixing yours. WTF?

Reply:
I fixed it. We now use a mix between the 2 options. Red lock in a circle for consistency, translucent background for more visibility.